js_modules/dagster-ui/packages/ui-core/src/settings/SettingsLeftPane.tsx

```diff
@@ -57,7 +57,7 @@ export const SettingsLeftPane = () => {
             <SideNavItem
               key={item.key}
               item={item}
-              active={item.type === 'link' && pathname === item.path}
+              active={item.type === 'link' && pathname.startsWith(item.path)}
```

Comment:
Is this necessary?

Reply:
I wanted the `Concurrency` tab to selected when viewing the detailed page.  The link item maps to the index page (without the specific key path).